dotnet-testing/MockingWithNSubstitute/MockingWithNSubstitute.Tests/NotificationServiceTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using FluentAssertions;
+using NSubstitute;
```

Comment:
These can also be moved to the global usings

Reply:
Sorted